_posts/2020-05-14-beginners-config.md

```diff
@@ -0,0 +1,116 @@
+
+
+## How to host a distributed Dask cluster
+
+The biggest jump for me was from running a local version of Dask for just an hour or so at a time during development, to standing up a production-ready version of Dask. Broadly there are two styles:
+
+1. a static dask cluster -- one that is always on, always awake, always ready to accept work
+2. an ephemeral dask cluster -- one that is spun up or down easily with a Python API, and, when on, starts a minimal dask master node that itself only spins up dask workers when work is actually submitted
+
+Though those are the two broad main categories, there are tons of choices of how to actually achieve that. It depends on a number of factors including what cloud provider products you want to use and if those resources are pre-provisioned for you and whether you want to use a python API or a different deployment tool to actually start the Dask processes. A very exhaustive list of all the different ways you could provision a dask cluster is in the dask docs under [Setup](https://docs.dask.org/en/latest/setup.html). As just a taste of what is described in those docs, you could:
+
+*   Install and start up the dask processes [manually from the CLI](https://docs.dask.org/en/latest/setup/cli.html) on cloud instances you provision, such as AWS EC2 or GCP GCE
+*   Use popular deployment interfaces such as [helm for kubernetes](https://docs.dask.org/en/latest/setup/kubernetes-helm.html) to deploy dask in cloud container clusters you provision, such as AWS Fargate or GCP GKE
+*   Use ‘native’ deployment python APIs, provided by the dask developers, to create (and interactively configure) dask on deployment infrastructure they support, namely kubernetes with [dask-kubernetes](https://kubernetes.dask.org/en/latest/) and YARN with [dask-yarn](https://yarn.dask.org/en/latest/), as long as you’ve already provisioned the kubernetes cluster or hadoop cluster already
+*   Use a nearly full-service deployment python API called [Dask Cloud Provider](https://cloudprovider.dask.org/en/latest/), that will go one step farther and provision the cluster for you too, as long as you give it AWS credentials (and as of time of writing, it only supports AWS)
+
+As you can see, there are a ton of options. On top of all of those, you might contract a managed service provider to provision and configure your dask cluster for you according to your specs, such as Saturn Cloud.
```

Comment:
Make Saturn Cloud a hyperlink to the website? I don't have any problems with that.